.github/workflows/master.yml

```diff
@@ -106,6 +107,9 @@ jobs:
         Rscript -e 'install.packages("h2o", repos="http://cran.r-project.org")'
     - name: Build package
       run: |
+        # Use r-devel on a release branch to run the same submission checks as CRAN.
```

Comment:
If we need `rocker` in order to test development versions of R, then we need to update `r-devel` to use `rocker`. However, our internal CI is incompatible with `rocker`, so we would then have to update internal CI to use another image. But, `Dockerfile.dev` also uses `rocker`, so we'd need to change `Dockerfile.dev` to use `r-base`, right?

Reply:
@dbczumar Thanks for the comment!

> our internal CI is incompatible with rocker.

Yep, can we do the following?

- In our internal CI, use `rocker:4.2.0` (which uses [ubuntu 20.04](https://github.com/rocker-org/rocker-versioned2/blob/master/dockerfiles/r-ver_4.2.0.Dockerfile)) and just build the package without running submission checks.
- In GitHub Actions, run submission checks using `rocker:r-devel`.

Reply:
Will `rocker/r-ver:4.2.0` switch to ubuntu 20.04 in the future? https://github.com/rocker-org/rocker-versioned2/issues/282#issuecomment-1108389687 says:

> Base images for R versions released before 2022-07-19 (i.e. 4.1.3, 4.2.0) will remain ubuntu:focal and will not switch.

Reply:
@eitsupi Thanks for the comment! Is there any other image than `rocker/r-ver:devel` that provides a development version of R?

Reply:
The Rocker project have `rocker/r-devel` and `rocker/drd`.
https://github.com/rocker-org/r-devel
https://github.com/rocker-org/drd

However, these are only built once a week, so if you want to try a newer version, `rocker/r-ver:devel`, which is built daily, is a good choice, I think.